README.md

```diff
@@ -131,12 +131,15 @@ List available reset and flashing plugins:
 $ htrun --plugins
 ```
 
-Flash binary file `/path/to/file/binary.bin` using plugin `stlink`. Use serial port `COM4` with baudrate `115200` to communicate with DUT:
+Ex: Flash binary file `/path/to/file/binary.bin` using STM32CubeProgrammer tool. Use serial port `COM4` with baudrate `115200` to communicate with DUT:
 
 ```
-htrun -c stlink -f /path/to/file/binary.bin -p COM4:115200
+htrun -c stprog -f /path/to/file/binary.bin -p COM4:115200
 ```
 
+NB: STM32CubeProgrammer should be installed first from https://www.st.com/en/development-tools/stm32cubeprog.html  
+and application path has to be in the environment path
```

Comment:
It doesn't make too much sense to say that a path has to be in the path.

```diff
- application path
+ application
```

Also, which application? The GUI one or the CLI one? Can we give the name of the binary/executable that must be on the path?

Reply:
done